server/reports/item-usage/2_5_0/argument_schemas/arguments.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
If we not changing API, don't need to update these

Reply:
API has changed to include stockOnHand rather than availableStockOnHand in item stats. 

However I think some fixes made to the 2.3.0 report which were made after I copied them into 2.5.0 - so I will add these fixes in now.